nevow.test.test_appserver.Logging.test_oldStyle fails

Against Twisted 14.0.0:

```
Traceback (most recent call last):
  File "/home/travis/build/twisted/nevow/nevow/test/test_appserver.py", line 187, in test_oldStyle
    proto = self.renderResource('/foo')
  File "/home/travis/build/twisted/nevow/nevow/test/test_appserver.py", line 170, in renderResource
    assert proto.transport.disconnecting
exceptions.AssertionError: 
```

--- a/nevow/test/test_appserver.py
+++ b/nevow/test/test_appserver.py
@@ -153,10 +153,10 @@ class Logging(testutil.TestCase):
             def renderHTTP(self, ctx):
                 return "boring"
         self.site = appserver.NevowSite(Res1())
+        self.site.startFactory()
+        self.addCleanup(self.site.stopFactory)
         self.site.logFile = StringIO()
 
-    def tearDown(self):
-        del self.site
 
     def renderResource(self, path):
         """@todo: share me"""
@@ -188,8 +188,11 @@ class Logging(testutil.TestCase):
         logLines = proto.site.logFile.getvalue().splitlines()
         self.assertEquals(len(logLines), 1)
         # print proto.transport.data.getvalue()
-        self.assertEquals(logLines,
-                          ['fakeaddress2 - - faketime "GET /foo HTTP/1.0" 200 6 "fakerefer" "fakeagent"'])
+        self.assertEquals(
+            logLines,
+            ['"fakeaddress2" - - faketime "GET /foo HTTP/1.0" 200 6 '
+             '"fakerefer" "fakeagent"'])
+
 
     def test_newStyle(self):
         class FakeLogger(object):

